common/utils/src/main/scala/org/apache/spark/internal/Logging.scala

```diff
@@ -158,7 +159,7 @@ trait Logging {
         }
       }
 
-      MessageWithContext(sb.toString(), context)
+      MessageWithContext(StringEscapeUtils.unescapeJava(sb.toString()), context)
```

Comment:
shall we change 
```
class LogEntry(messageWithContext: => MessageWithContext) {
  def message: String = messageWithContext.message
```
instead? 
There are many log concations. We should avoid calling the method `unescapeJava` for all the log fragments.

Reply:
Updated.